terraform/aws/analytical-platform-data-production/airflow/iam-policies.tf

```diff
@@ -145,3 +146,108 @@ data "aws_iam_policy_document" "airflow_dev_cluster_autoscaler_assume_role_polic
     actions = ["sts:AssumeRole"]
   }
 }
+
+# flow log policy
+
+data "aws_iam_policy_document" "airflow_dev_flow_log_role_policy" {
+  statement {
+    sid    = ""
+    effect = "Allow"
+    actions = [
+      "logs:PutLogEvents",
+      "logs:DescribeLogStreams",
+      "logs:DescribeLogGroups",
+      "logs:CreateLogStream",
+      "logs:CreateLogGroup"
+    ]
+    resources = ["*"]
+  }
+
+}
+
+
+
+data "aws_iam_policy_document" "airflow_dev_flow_log_assume_policy" {
+  statement {
+    sid    = ""
+    effect = "Allow"
+    principals {
+      type = "Service"
+      identifiers = [
+        "vpc-flow-logs.amazonaws.com"
+      ]
+    }
+    actions = ["sts:AssumeRole"]
+    condition {
+      test     = "StringEquals"
+      variable = "aws:SourceAccount"
+      values   = ["593291632749"]
+    }
+
+    condition {
+      test     = "ArnLike"
+      variable = "aws:SourceArn"
+      values   = ["arn:aws:ec2:eu-west-1:593291632749:vpc-flow-log/*"]
+    }
+  }
+
+}
+
+# airflow dev node instance policy
+
+data "aws_iam_policy_document" "airflow_dev_node_instance_inline_role_policy" {
+  statement {
+    sid    = ""
+    effect = "Allow"
+    principals {
+      type = "Service"
+      identifiers = [
+        "arn:aws:iam::593291632749:role/airflow-dev-cluster-autoscaler-role",
+        "arn:aws:iam::593291632749:role/airflow*"
+      ]
+    }
+    actions = ["sts:AssumeRole"]
+  }
+
+}
+
+
+data "aws_iam_policy_document" "airflow_dev_node_instance_assume_role_policy" {
```

Comment:
This doesn't look like the right trust policy? The node instance role should only be assumable by EC2

Reply:
Yeah this is just a copy paste of a different trust policy, so will not work properly as is